libshaderc/include/shaderc.h

```diff
@@ -283,6 +292,10 @@ size_t shaderc_module_get_num_warnings(const shaderc_spv_module_t module);
 // Returns the number of errors generated during the compilation.
 size_t shaderc_module_get_num_errors(const shaderc_spv_module_t module);
 
+// Returns the type of the generated compilation errors. Returns
```

Comment:
This could be combined with shaderc_module_get_success into shaderc_module_get_compilation_result.